lib/hil/hil.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+import requests
+import sys
+import yaml
+
```

Comment:
@vsemp this file provides many useful wrapper functions to abstract the rest calls to the HIL api. However, i'm not sure we want to add another cli to QUADS, let alone one that is specific to HIL. Maybe we could get rid of the command line parsing, and add these as private methods when we implement the concrete HIL inventory class?